test/expected/agg_bookends.out

```diff
@@ -0,0 +1,110 @@
+\ir include/create_single_db.sql
+SET client_min_messages = WARNING;
+DROP DATABASE IF EXISTS single;
+SET client_min_messages = NOTICE;
+CREATE DATABASE single;
+\c single
+CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
+psql:include/create_single_db.sql:7: NOTICE:  installing required extension "postgres_fdw"
+SELECT setup_timescaledb(hostname => 'fakehost'); -- fakehost makes sure there is no network connection
+ setup_timescaledb 
+-------------------
+ 
+(1 row)
+
+CREATE TABLE "btest"(time timestamp, gp INTEGER, temp float, strid TEXT DEFAULT 'testing');
+SELECT create_hypertable('"btest"', 'time');
+ create_hypertable 
+-------------------
+ 
+(1 row)
+
+INSERT INTO "btest" VALUES('2017-01-20T09:00:01',1, 22.5);
```

Comment:
Can you add another "time" column and test last() when alternating between, e.g., "time1" and "time2", which have different last() values?

Also, if you can do the thing suggested above with defaulting to hypertable time-column, I'd like to see tests that show the default and non-default columns being used.